model/src/main/kotlin/Hash.kt

```diff
@@ -63,12 +68,20 @@ data class Hash(
         }
     }
 
+    private val invalidAlgorithms = setOf(HashAlgorithm.NONE, HashAlgorithm.UNKNOWN)
+    private val validAlgorithms = enumValues<HashAlgorithm>().toSet() - invalidAlgorithms
```

Comment:
What about `unverifiableAlgorithms` and `verifiableAlgorithms` instead?

Reply:
I was actually thinking about these, but found them to sound too awkward. But anyway, as you mention them and it better matches the verify functions, I'll make the change.